pkg/pillar/zedcloud/send.go

```diff
@@ -362,11 +365,13 @@ func VerifyAllIntf(ctx *ZedCloudContext,
 // to allow the caller to look at StatusCode
 // We return a SenderResult enum for various error cases.
 // the controller but it is overloaded, or has certificate issues.
+// The caller is responsible to handle any required AuthContainer by calling
```

Comment:
Was the commit message not clear?
"This prepares for being able to checkpoint the received configuration
with its AuthContainer wrapper."

Then the implementation of that is in the second commit.

Reply:
Oh, it seems I misunderstood the logic of function `verifyAuthentification`, sorry. The name confuse me because the logic inside extracts payload, not only does verification. Perhaps it will make sense to rename the function to VerifyAndExtractAuthContainer or something like this?

Reply:
I'll rename it to RemoveAndVerifyAuthContainer()